tutorials/500_spatiotemporal.ipynb

```diff
@@ -7,23 +7,43 @@
    "source": [
```

Comment:
<p>Don't we want to use <code>score_genes_for_marginals</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/moscot_notebooks/pull/45/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>i think we can avoid it here and keep the results as is. We have other examples using it.</p>

Reply:
<p>and the reproducibility repo to recover results</p>

Reply:
The results we show in the paper make use of it. So I would also add it here tbh.